clang/test/CodeGenSYCL/add_ir_attributes_global_variable.cpp

```diff
@@ -81,6 +95,10 @@ int main() {
 // CHECK-DAG: @_ZL3h_v = internal addrspace(1) constant %struct.h { {{.*}} }, {{.*}} #[[GlobalVarHAttrs:[0-9]+]]
 // CHECK-DAG: @_ZL4gh_v = internal addrspace(1) constant %struct.gh { {{.*}} }, {{.*}} #[[GlobalVarHGAndGHAttrs:[0-9]+]]
 // CHECK-DAG: @_ZL4hg_v = internal addrspace(1) constant %struct.hg { {{.*}} }, {{.*}} #[[GlobalVarHGAndGHAttrs]]
+// CHECK-DAG: @_ZL4ig_v = internal addrspace(1) constant {{.*}}, align 4{{$}}
```

Comment:
Shouldn't this have attributes? Since type `ig` is instantiated as:

`ig<prop1, prop2, prop3, prop4, prop5, prop6, prop7> ig_v`

Reply:
Oh right. Sorry. Got confused for a second.